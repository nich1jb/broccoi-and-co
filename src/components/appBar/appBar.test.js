import React from 'react';
import { render, cleanup } from '@testing-library/react';
import "@testing-library/jest-dom/extend-expect";
import AppBar from '.';

afterEach(cleanup);

describe('AppBar', () => {
  it("matches snapshot for header", () => {
    const { asFragment } = render(<AppBar position='header' />);
    expect(asFragment()).toMatchSnapshot();
  });
  
  it("matches snapshot for footer", () => {
    const { asFragment } = render(<AppBar position='footer' />);
    expect(asFragment()).toMatchSnapshot();
  });
})