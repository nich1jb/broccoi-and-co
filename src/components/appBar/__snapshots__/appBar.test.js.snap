// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AppBar matches snapshot for footer 1`] = `
<CustomAppBar
  position="footer"
>
  <Text>
    Made with 
    <span
      aria-label="heart"
      role="img"
    >
      ðŸ’š
    </span>
     in Melbourne
  </Text>
  <Text>
    Â© 2016 Broccoli & Co. All Rights Reserved
  </Text>
</CustomAppBar>
`;

exports[`AppBar matches snapshot for header 1`] = `
<CustomAppBar
  position="header"
>
  <LogoImg
    src="images/logo.png"
  />
</CustomAppBar>
`;
